### ISSUE TITLE
Mistake:  ' aria-live="true" ' is not a valid value

### ISSUE THREAD

**Issue by IpsumLorem16** [google/WebFundamentals#6936]
**Page Affected:**
https://developers.google.com/web/fundamentals/accessibility/semantics-aria/hiding-and-updating-content
**What needs to be done?**
You say:
>**aria-live** has three allowable values: **polite, assertive, and off.**  

But then go on to say:
> aria-atomic indicates whether the entire region should be considered as a whole when communicating updates. For example, if a date widget consisting of a day, month, and year has **aria-live=true** and aria-atomic=true

Is this not a mistake? aria-live="true" is not one of the three allowed values.
https://www.w3.org/TR/wai-aria-1.1/#aria-live
